content/en/docs/release-oversight/testing-in-production.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Testing in \"production\""
+description: Running our development tests on our latest stable sprint release.
+---
+
+### Background
+Like security, quality comes in layers.  In addition to the direct signal from our CI, [release gate testing](/docs/architecture/release-gating/) and Quality Engineering, we're now testing our sprint output on a critical piece of our own infrastructure.
+
+### Our philosophy on failure
+It's important to understand we believe breaking ourselves first is a form of success.  Failing a sprint is a gift in this sense.  It gives us time to adjust before our next big release.  What this means in practice is that, when sprints fail, we commit to clearing our sprint of feature work and focusing on stability across whatever teams necessary.
+
+### Sprint success or failure
+There are couple ways we judge the success or a failure of a sprint:
+
+* Did we have a build on time?
+* Were any changes required by the administrator not covered by an [admin ack](https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare)?
+* Did we find a significant problem that forced us to remove the production cluster from rotation?
+
+### Selecting the build
+The selection process is quite simple.  On the last Wednesday of each sprint the [Technical Release Team](/docs/release-oversight/the-technical-release-team/) (TRT) will select an accepted payload from the last two days.  If no payload is readily available and time allows for deeper investigation, the TRT may manually accept a payload.  This involves carefully accounting for extraneous circumstances that can lead to widespread payload blockage.  Cloud provider or other infrastructure related problems are just two examples.
+
+From there Quality Engineering will run additional tests.  Barring any new surprises Test Platform will be notified that a release is ready.
+
+It's worth noting that we'll likely create a dedicated channel for managing these releases through the [Update Service](https://docs.openshift.com/container-platform/4.7/updating/understanding-the-update-service.html) in the upcoming months.
```

Comment:
A new channel should be an enhancement pull request as it has some impact on overall OpenShift user experience.

Reply:
I agree.  If you'd rather I not mention this I can remove it.  I was mostly responding to the questions I've received.  I did change the "we'll likely" to "we may" to make it sound more tentative.